alica_standard_library/include/GenerateRandom.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <engine/BasicBehaviour.h>
+
+#include <boost/dll/alias.hpp>
+
+namespace alica_standard_library
+{
+
+class GenerateRandom : public alica::BasicBehaviour
+{
+public:
+    GenerateRandom(alica::BehaviourContext& context);
+    void initialiseParameters() override;
+    static std::unique_ptr<GenerateRandom> create(alica::BehaviourContext& context);
```

Comment:
idk if it's the goal of this PR but I see that this create method is repeated everytime, can we put this to the base class templated somehow?

Reply:
I think since this needs to be static, anything in a base class won't help too much.  This function will be imported using https://github.com/rapyuta-robotics/alica/blob/5641acf8f1b85ee441f3258d0a0bdc52442fb0ac/alica_dynamic_loading/src/DynamicBehaviourCreator.cpp#L28 which does not have access to the type of the return value, only the exported symbol name.  So I don't think we can get around creating a function of the appropriate signature.  There may be a clever trick but I don't have any ideas.

Reply:
Can use CRTP pattern like I had done in lbc. Maybe I can add it as a util in alica itself & can use that everywhere